README.md

```diff
@@ -50,6 +50,15 @@ If you would like to contribute, please, fork the repo and open a Pull Request (
 
 **Thank you!**
 
+## ⚙️ Components ⚙️
+
+RustWorks is composed of several components which are listed here:
+- [rcas](https://github.com/nw-rs/rcas) - Open source computer algebra system
+- [board-support](https://github.com/nw-rs/board-support) - Files for compatibility with NumWorks n0110 hardware
```

Comment:
Should be "Drivers for the calculator's hardware. Currently only supports NumWorks n0110."